react_frontend/src/components/SearchBar.js

```diff
@@ -0,0 +1,74 @@
+import '../assets/SearchBar.css'
+/* import React, { Component } from 'react'

```

Comment:
LGTM but might want to reduce the number of comments 👍🏾

Reply:
Thanks for pointing that out, I left it because I thought I was going to use some of the commented out code for future tasks.

Reply:
I will try to keep comments organized if there are any for future pull requests.

Reply:
Sounds good! great work harumi!

Reply:
thanks man